libraries/engage/back-in-stock/components/Subscriptions/components/Subscription/index.jsx

```diff
@@ -0,0 +1,181 @@
+    marginTop: '8px',
+  }).toString(),
+  priceContainer: css({
+    minWidth: 100,
+  }).toString(),
+  priceInfo: css({
+    wordBreak: 'break-word',
+    fontSize: '0.875rem',
+    lineHeight: '0.875rem',
+    color: 'var(--color-text-low-emphasis)',
+    padding: `${variables.gap.xsmall}px 0`,
+    textAlign: 'right',
+  }).toString(),
+  titleContainer: css({
+  }).toString(),
+  title: css({
+    fontSize: 17,
+    fontWeight: 600,
+    flexWrap: 'wrap',
+    overflowWrap: 'anywhere',
+  }).toString(),
+  removeContainer: css({
+    minWidth: '30px',
+  }).toString(),
+  availabilityText: css({
+    fontSize: '0.875rem',
+  }).toString(),
+};
+
+/**
+ * Renders on single Back in Stock subscription
+ * @param {Object} props Props.
+ * @param {string} props.subscription The subscription which should be rendered
+ * @returns {JSX}
+ */
+const Subscription = ({
+  subscription,
+}) => {
+  const { subscriptionCode, product } = subscription;
+  const {
+    removeBackInStockSubscription,
+  } = useBackInStockSubscriptions();
+  const { ListImage: gridResolutions } = getThemeSettings('AppImages') || {};
+  const currency = product.price?.currency || 'EUR';
+  const defaultPrice = product.price?.unitPrice || 0;
+  const specialPrice = product.price?.unitPriceStriked;
+  const hasStrikePrice = product.price?.discount > 0;
+  const productLink = getProductRoute(product.id);
+
+  return (
+    <div className={styles.root}>
+      <Link
+        className={styles.imageContainer}
+        component="div"
+        href={productLink}
+        aria-hidden
+      >
+        <ProductImage src={product.featuredImageBaseUrl} resolutions={gridResolutions} />
+      </Link>
+
+      <div className={styles.infoContainer}>
+        <div className={styles.infoContainerRow}>
+          <div>
+            <Link
+              href={productLink}
+              tag="span"
+              className={styles.titleContainer}
+            >
+              <span
+                className={styles.title}
+                // eslint-disable-next-line react/no-danger
+                dangerouslySetInnerHTML={{ __html: `${product.name}` }}
+              />
+            </Link>
+          </div>
+          <div className={styles.removeContainer}>
+            <button
+              className={styles.root}
+              onClick={() => removeBackInStockSubscription({ subscriptionCode })}
+              type="button"
+              aria-label={i18n.text('favorites.remove')}
+            >
+              <Ripple className={styles.ripple}>
+                <CrossIcon />
+              </Ripple>
+            </button>
+          </div>
+        </div>
+        <div className={classNames(styles.baseContainerRow)}>
+          <AvailableText
+            text={product?.availability?.text}
+            state={product?.availability?.state}
+            showWhenAvailable={false}
+            className={styles.availabilityText}
+          />
+          <BackInStockButton
```

Comment:
why is this right aligned? since it relates to the availability text, should it be aligned like that? i think at the other places where this component is used that's the case.

![Bildschirmfoto 2024-02-26 um 13 42 00](https://github.com/shopgate/pwa/assets/13087792/b04f5e56-af38-49ce-bdb8-456f2afe1f3a)

changing this would give enough space at the right side to move the prices up and remove  some whitespace

Reply:
now the button is misaligned within the characteristics sheet
![Bildschirmfoto 2024-02-29 um 08 43 22](https://github.com/shopgate/pwa/assets/13087792/ffa9f211-59fb-4d99-8943-bcff321792de)